tutorials/smart-contracts/launch-your-first-project/test-and-deploy-with-hardhat.md

```diff
@@ -69,7 +69,7 @@ Let's start by setting up Hardhat for your Storage contract project:
     The configuration also defines two network settings: 
 
     - `localNode` - runs a PolkaVM instance on `http://127.0.0.1:8545` for local development and testing
-    - `westendAssetHub` - connects to the Westend Hub network using a predefined RPC URL and a private key stored in environment variables
+    - `passetHub` - connects to the Passet Hub network using a predefined RPC URL and a private key stored in environment variables
```

Comment:
🚩

Reply:
These are the only remaining references to passetHub. Some network configs in hardhat and scripts still use this name, but there are explanations mentioning it is the Polkadot Hub TestNet